lib/tenableio_scan_handler.py

```diff
@@ -7,22 +7,31 @@
 from lib.response import Response
 from lib.hosts import Hosts
 from lib.event import Event
+from scanners.tenable_io_scanner import TIOScanner
+# from lib.custom_exceptions import TenableScanCompleteException
+from lib.s3_helper import send_to_s3
+
+S3_CLIENT = boto3.client('s3', region_name='us-west-2')
```

Comment:
Should our regions be configurable or is this just the default if a region isn't specified?

Reply:
I don't think it's the default, we are forcefully specifying it.
The reasons are:
- Everything about vautomator is currently deployed in `us-west-2`
- Some time ago, AWS gurus in the team asked people to use 

In saying this, it is not very configurable though, yeah. I will add an environment variable in `serverless.yml` for region, and use that in here to make it more configurable.